cmake/FindWasmtime.cmake

```diff
@@ -0,0 +1,15 @@
+find_path2(WASMTIME_INCLUDE_DIR wasmtime.h
+  PATHS ${DEPS_PREFIX}/../build/src/wasmtime/crates/c-api/include
```

Comment:
TODO(dundargoc): hacky, should add a directory for the `.deps` directory directly, alternatively ensure the relevant files are in usr.